GitUI/Editor/FileViewer.cs

```diff
@@ -68,6 +68,10 @@ public partial class FileViewer : GitModuleControl
         [Browsable(false)]
         public byte[] FilePreamble { get; private set; }
 
+        [Description("Show syntax highlighting in diffs.")]
+        [DefaultValue(true)]
+        public bool ShowSyntaxHighlightingInDiff { get; set; }
```

Comment:
Do we want this status variable?
I first thought AppSettings.ShowSyntaxHighlightingInDiff could be used, but that may not be stored.
showSyntaxHighlighting.Checked is still local

Reply:
It is there to be consistent with the pattern used for each of the settings in the FileViewer toolbar.

Reply:
I haven't reviewed all other properties, but this is very leaky. The property could be set from code and it won't be updating user settings; which can create problems.

This property doesn't appear to be used outside the class, so there doesn't appear necessary to expose it.
Please convert it to a private field.
For the tests it can be exposed via the `TestAccessor`.